tests/web/test_assets.py

```diff
@@ -81,3 +83,62 @@ def test_get_assets_limit(client, b):
         res = client.get(ASSETS_ENDPOINT + '?search=abc&limit=1')
         assert res.status_code == 200
         assert len(res.json) == 1
+
+
+@pytest.mark.bdb
+@pytest.mark.tendermint
+def test_get_assets_tmint(client, tb):
```

Comment:
Minor comment: I'd call it `test_get_asset_tendermint`, why? Less things to process for your brain :) (I asked myself "why tmint and not tendermint?")